repl/src/main/scala/org/apache/spark/repl/ExecutorClassLoader.scala

```diff
@@ -44,7 +44,7 @@ class ExecutorClassLoader(
     env: SparkEnv,
     classUri: String,
     parent: ClassLoader,
-    userClassPathFirst: Boolean) extends ClassLoader with Logging {
+    userClassPathFirst: Boolean) extends ClassLoader(null) with Logging {
```

Comment:
Maybe add comment over this to explain why we are setting a `null` value to parent?